src/scene/mesh-instance.js

```diff
@@ -48,7 +50,7 @@ class Command {
  * @classdesc An instance of a {@link pc.Mesh}. A single mesh can be referenced by many
  * mesh instances that can have different transforms and materials.
  * @description Create a new mesh instance.
- * @param {pc.GraphNode} node - The graph node defining the transform for this instance.
+ * @param {pc.GraphNode} [node] - The graph node defining the transform for this instance. This parameter is optional when used with {@link pc.RenderComponent} and will use the node the component is attached to.
```

Comment:
this is failing .. need to find different solution.
"Error: build/playcanvas.d.ts(13658,42): error TS1016: A required parameter cannot follow an optional parameter."

Reply:
done